examples/quickstart/README.md

```diff
@@ -0,0 +1,87 @@
+# Example setup
+
+This example shows the bare minimum requirements to deploy applications on wasmCloud.
+
+It relies on the Kubernetes `default` namespace for simplicity.
+
+## Install [NATS](https://github.com/nats-io/nats-server)
+
+```bash
+helm repo add nats https://nats-io.github.io/k8s/helm/charts/
+helm upgrade --install -f nats-values.yaml nats-cluster nats/nats
+```
+
+Validate installation with:
+
+```bash
+# make sure pods are ready
+kubectl get statefulset,deployment -l app.kubernetes.io/instance=nats-cluster
+```
+
+## Install wasmCloud Application Deployment Manager - [wadm](https://github.com/wasmCloud/wadm)
+
+```sh
+helm install wadm -f wadm-values.yaml --version 0.2.0 oci://ghcr.io/wasmcloud/charts/wadm
```

Comment:
I think it's probably best not to explicitly include a `--version` here so that the latest version of the chart gets used
```suggestion
helm install wadm -f wadm-values.yaml oci://ghcr.io/wasmcloud/charts/wadm
```